sdk/servicebus/azure-servicebus/stress/scripts/conftest.py

```diff
@@ -0,0 +1,46 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+import pytest
+import os
+import sys
+from azure.servicebus import ServiceBusClient
+from dotenv import load_dotenv
+
+LOGGING_ENABLE = False
+ENV_FILE = os.environ.get('ENV_FILE')
+load_dotenv(dotenv_path=ENV_FILE, override=True)
+
+# Allow us to pass stress_test_duration from the command line. TODO: may not actually be using this.
+def pytest_addoption(parser):
+    parser.addoption('--stress_test_duration_seconds', action="store", default=None)
```

Comment:
Given the typical duration of stress tests (hours or even days!) you might want to consider a _slightly_ larger default unit (maybe minutes?)

Reply:
noted for future changes

Reply:
A polite way of saying 'we're not doing this'. I can accept this.

Reply:
nooo, I just meant noted so I can make the change this upcoming sprint :)